be/src/olap/rowset/segment_v2/encoding_info.h

```diff
@@ -31,8 +34,6 @@ namespace segment_v2 {
 
 class PageBuilder;
 class PageDecoder;
```

Comment:
forward declaration of PageBuilder and PageDecoder could also be removed since we've included the header